common/source/docs/common-remoteid.rst

```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
 OpenDroneID
 ===========
 
+.. image:: ../../../images/remoteid.jpg
+    :target: ../../_images/remoteid.jpg
+
+
 ArduPilot provides support for OpenDroneID in firmware versions 4.2 and later via MAVLink to an OpenDroneID compatible module attached to either a serial port or DroneCAN port on the autopilot.
 
 In order to provide tamper protection, a special autopilot firmware must be built and loaded on the autopilot which: enables this feature, prevents loading non-compliant firmware via GCS or normal uploaders, and provides further tamper-proofing.
```

Comment:
Make this a bullet list perhaps

Reply:
its not a series of steps or things to do....general info